File: Python/Pascal's Triangle/Pascal_triangle.py
  
def Pascal(n):  
  
    for l in range(1, n + 1):  
        x = 1; 
        for i in range(1, l + 1):  
              
            
            print(x, end = " ");  
            x = int(x * (l - i) / i);  
        print("");  
n=int(input())
print (Pascal(n))

File: Javascript/PascalTriangle/pascalTriangle.js
const readline = require('readline');
let numRows;

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
});

rl.question("Enter the number of rows of the Pascal's Triangle: ", answer => {
  numRows = answer;

  const generateTriangle = numRows => {
    let pascalTriangle = new Array(numRows);

    for (let i = 0; i < numRows; i++) {
      const row = new Array(i + 1);

      row[0] = 1;
      row[row.length - 1] = 1;

      for (let j = 1; j < row.length - 1; j++) {
        let rowAbove = pascalTriangle[i - 1];
        row[j] = rowAbove[j] + rowAbove[j - 1];
      }

      pascalTriangle[i] = row;
      console.log(pascalTriangle[i]);
    }
    return pascalTriangle;
  };
  generateTriangle(numRows);
  rl.close();
});
